ddtrace/internal/telemetry/writer.py

```diff
@@ -318,6 +324,14 @@ def _app_started_event(self):
                 (TELEMETRY_TRACE_SAMPLING_LIMIT, config._trace_rate_limit, "unknown"),
                 (TELEMETRY_SPAN_SAMPLING_RULES, config._sampling_rules, "unknown"),
                 (TELEMETRY_SPAN_SAMPLING_RULES_FILE, config._sampling_rules_file, "unknown"),
+                (TELEMETRY_TRACE_SPAN_ATTRIBUTE_SCHEMA, SCHEMA_VERSION, "unknown"),
+                (TELEMETRY_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED, _remove_client_service_names, "unknown"),
+                (
+                    TELEMETRY_TRACE_PEER_SERVICE_DEFAULTS_ENABLED,
+                    os.getenv("DD_TRACE_PEER_SERVICE_DEFAULTS_ENABLED", default=False),
+                    "unknown"
+                ),
+                (TELEMETRY_TRACE_PEER_SERVICE_MAPPING, os.getenv("DD_TRACE_PEER_SERVICE_MAPPING", default=""), "unknown"),
```

Comment:
We should avoid reading environment variables here. We should try to use the existing pattern of storing environment variables in a config object (via envier or ddtrace.config) and then reading the configuration object here.